blocks/includes/blocks/class-urlslab-faqs.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class Urlslab_FAQs extends Urlslab_Gutenberg_Block {
```

Comment:
fix all names of classes 
Correct classname should be e.g.  Urlslab_Blocks_Faqs